source/extensions/geoip_providers/maxmind/geoip_provider.cc

```diff
@@ -72,7 +72,7 @@ GeoipProviderConfig::GeoipProviderConfig(
                            : absl::nullopt;
   isp_header_ = !geo_headers_to_add.isp().empty() ? absl::make_optional(geo_headers_to_add.isp())
                                                   : absl::nullopt;
-  apple_private_relay_header_ = !geo_headers_to_add.isp().empty()
+  apple_private_relay_header_ = !geo_headers_to_add.apple_private_relay().empty()
```

Comment:
i think this change is ok for alpha extension state (is a potentially breaking change for users).